### PR TITLE
It should be possible to split col or row spanning cell which has cursor but not cell-selection.

### PR DESCRIPTION
**Pull request by jpuri**
Currently its not possible to split spanning cell with cursor in it. PR fixes this issue.

### CODE REVIEW

File: test/test-commands.js [hunk 1]
```diff
@@ -317,11 +317,16 @@ describe("mergeCells", () => {
 })
 
 describe("splitCell", () => {
-  it("does nothing when there isn't a cell selection", () =>
+  it("does nothing when there isn't a cell with cursor is not spaning", () =>
```
[the remark this answers is not in the document]
Comment (jpuri) [reply test/test-commands.js]: This is existing test, I only changed description, I can change it to what you described above.

File: src/util.js [hunk 2]
```diff
@@ -12,6 +12,12 @@ export function cellAround($pos) {
   return null
 }
 
+export function cellWrapping($pos) {
```
[the remark this answers is not in the document]
Comment (jpuri) [reply src/util.js]: Let me check

[the remark this answers is not in the document]
Comment (jpuri) [reply src/util.js]: `cellAround` returns resolved pos for `row`.
`cellAround` returns enclosing cell node.

**Comment by jpuri** [comment]
PR is updated @eshvedai

File: src/commands.js [hunk 3]
```diff
@@ -275,9 +276,17 @@ export function mergeCells(state, dispatch) {
...
 export function splitCell(state, dispatch) {
   let sel = state.selection
-  if (!(sel instanceof CellSelection) || sel.$anchorCell.pos != sel.$headCell.pos) return false
-  let cellNode = sel.$anchorCell.nodeAfter
-  if (cellNode.attrs.colspan == 1 && cellNode.attrs.rowspan == 1) return false
+  let cellNode, oldPos
+  if (!(sel instanceof CellSelection)) {
+    cellNode = cellWrapping(sel.$from)
+    if (!cellNode) return false
+    oldPos = cellAround(sel.$from).pos
```
Comment (eshvedai) [inline src/commands.js]: `oldPos` is the current cell position, right? lets call it  `cellPos`?

**Comment by jpuri** [comment]
PR updated again.

**Pull request merged**